modular_skyrat/modules/importedvendors/code/vendor_snacks.dm

```diff
@@ -0,0 +1,275 @@
+// Snacks and drinks for the 'snacks' tab of vendors
+
+//Yangyu Stuff
+
+/obj/item/reagent_containers/cup/glass/dry_ramen/prepared
+	name = "cup ramen"
+	desc = "This one even comes with water, amazing!"
+	list_reagents = list(/datum/reagent/consumable/hot_ramen = 15, /datum/reagent/consumable/salt = 3)
+
+/obj/item/reagent_containers/cup/glass/dry_ramen/prepared/hell
+	name = "spicy cup ramen"
+	desc = "This one comes with water, AND a security checkpoint's worth of capsaicin!"
+	list_reagents = list(/datum/reagent/consumable/hell_ramen = 15, /datum/reagent/consumable/salt = 3)
+
+/obj/item/food/vendor_snacks
+	name = "\improper God's Strongest Snacks"
+	desc = "You best hope you aren't a sinner. (You should never see this item please report it)"
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_quick_foods.dmi'
+	icon_state = "foodpack_generic"
+	trash_type = /obj/item/trash/vendor_trash
+	bite_consumption = 10
+	food_reagents = list(/datum/reagent/consumable/nutriment = INFINITY)
+	junkiness = 10
+	custom_price = PAYCHECK_LOWER * INFINITY
```

Comment:
Same as above, scared to know what Infinity could do to the game systems, better to make it a simple joke and leave it

Reply:
be not afraid of god's strongest snacks